data/vendors/jamf.toml

```diff
@@ -0,0 +1,11 @@
+name = "JAMF"
```

Comment:
Could you clarify a bit on which Jamf component this relates to if it's not all of them?